src/client/cypress/e2e/editor/lithologicalDescription.cy.js

```diff
@@ -0,0 +1,214 @@
+import { newEditableBorehole } from "../testHelpers";
+
+describe("Tests for the lithological description column.", () => {
+  it("Creates, updates and deletes lithological descriptions ", () => {
+    // create boreholes
+    newEditableBorehole().as("borehole_id");
+
+    // navigate to stratigraphy
+    cy.get('[data-cy="stratigraphy-menu-item"]').click();
+    cy.get('[data-cy="add-stratigraphy-button"]').click();
+    cy.wait("@stratigraphy_edit_create");
+
+    // try add lithological description without lithology
+    cy.get('[data-cy="add-litho-desc-icon"]').click();
+    cy.wait("@lithological_description");
+
+    //Assure cannot add lithological description without lithology.
+    cy.get('[data-cy="lithological-description-0"]').should("not.exist");
+
+    // Add three layers
```

Comment:
PP: Alle Kommentare gleich schreiben: Mit Abstand nach dem `//` und dann entweder durchwegs mit Klein- oder Grossbuchstaben beginnen. In JavaScript fangen wir üblich mit Kleinbuchstaben an.